staging/src/k8s.io/noderesourcetopology-api/pkg/apis/topology/v1alpha1/types.go

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
API types for a k8s.io API need to located in a directory with an OWNERS file that looks something like https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/OWNERS, to make sure changes hit API review

it's not clear to me whether the types.go files or the crd.yaml file is authoritative for this API

Reply:
Thanks for pointing this out. I will add an OWNERS file so changes hit API review. I think types.go should be considered authoritative source here. Major changes in types.go would need most likely need updates to crd.yaml anyway.